client/src/components/views/inventory items/Parts.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment, useState } from 'react';
+import { Table, Thead, Tbody, Tr, TableCaption, Heading, Image } from '@chakra-ui/react';
+import Loader from 'components/common/Loader';
+import { getParts } from 'utils/api/parts.js';
+import { useQuery } from 'react-query';
+import { StyledTableRow, StyledTableHeader, StyledTableCell } from '../../common/Table';
+import { TablePagination } from '@material-ui/core';
+
+const Parts = () => {
+  const { isLoading, isSuccess, data } = useQuery('parts', getParts);
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  if (isSuccess && data.data.length === 0) {
+    return (
+      <>
+        <Heading size="xl" textAlign="center" mt={5}>
+          No Parts
+        </Heading>
+        <Image
+          src="/images/noResults.png"
+          alt="No results illustration"
+          width="100%"
+          height="100%"
+        />
```

Comment:
Now that we are using this Image component in 4 different places, we could extract it into a common place.

And I feel that 100% might be too big?

Reply:
you're right, will look into refactoring

Reply:
next sprint gordo